src/main/content/_assets/js/guide-common.js

```diff
@@ -453,6 +453,15 @@ $(document).ready(function () {
         $(this).css("opacity", "1");
     });
 
+    // set up html for copy code block accessibility
+    var static_codeblock_selector = "#guide_content pre:not(.no_copy pre):not(.code_command pre):not(.hotspot pre):not(.code_column pre), #guide_content .code_command .content"
+    $(static_codeblock_selector).each(function (){
+        $(this).wrap('<div class="code_block_wrapper" title="Code block"></div>');  
+    })
+    $('.code_block_wrapper').each(function (){
+        $(this).prepend('<div id="copied_confirmation">Copied to clipboard</div><input type="image" id="copy_to_clipboard" src="/img/guides_copy_button.svg" alt="Copy code block" title="Copy code block"/>');
```

Comment:
Can we change the id of copy_to_clipboard to a class, since there could be many instances of the same id, especially on the start page?